packages/ui/src/components/DataStory/clients/WorkspaceApiClient.tsx

```diff
@@ -0,0 +1,63 @@
+import { core, Diagram, multiline, NodeDescription, nodes } from '@data-story/core';
+import { WorkspacesApi } from './WorkspacesApi';
+
+export const createDiagram = (content = 'Diagram') => {
+  const { Signal, Comment, Ignore } = nodes;
+
+  const diagram = core.getDiagramBuilder()
+    .add({...Signal, label: 'DataSource'}, { period: 200, count: 100})
+    .add({...Ignore, label: 'Storage'})
+    .above('Signal.1').add(Comment, { content: `### ${content} 🔥`})
+    .get();
+
+  return diagram;
+}
+
+export class WorkspaceApiClient {
+  workspacesApi: WorkspacesApi = {
+    getNodeDescriptions: async({ path }) => {
+      return [] as NodeDescription[]
+    },
+    getTree: async({ path }) => {
+      // const treeJson = localStorage.getItem(path)
+      // console.log('treeJson', treeJson)
+      // if(treeJson) return parseDiagramTree(treeJson)
+
+      // console.log('No tree found at path', path)
+      // If no tree at path
+      // For testing purposes: Persist and return a default tree
+      const defaultTree = [{
+        path: '/',
+        type: 'folder',
+        name: '/',
+        id: 'root',
```

Comment:
Does id = 'root' play a role, or is it an arbitrary name? I see the other ids are corresponding to path, without slashes.

Reply:
The `id` should serve as a unique identifier within react-arborist, with the option to use slashes for separation. If you find the `path` and `id` to be too redundant, I would suggest considering the use of `idAccessor` as a replacement.
https://github.com/brimdata/react-arborist?tab=readme-ov-file#data-with-different-property-names

Reply:
Lets keep it, if needed we reconsider details later 👍